ext/storage_sources/gcp_store/gcp_storage_source.cpp

```diff
@@ -292,7 +393,37 @@ gcp_rename(WT_FILE_SYSTEM *file_system, WT_SESSION *session, const char *from, c
 }
 
 static int
-gcp_file_size(WT_FILE_SYSTEM *file_system, WT_SESSION *session, const char *name, wt_off_t *sizep)
+gcp_file_lock(WT_FILE_HANDLE *file_handle, WT_SESSION *session, bool lock)
+{
+    // Locks are always granted.
+    WT_UNUSED(file_handle);
+    WT_UNUSED(session);
+    WT_UNUSED(lock);
+
+    return 0;
+}
+
+static int
+gcp_file_size(WT_FILE_HANDLE *file_handle, WT_SESSION *session, wt_off_t *sizep)
+{
+    WT_UNUSED(session);
+
+    gcp_file_handle *gcp_fh = reinterpret_cast<gcp_file_handle *>(file_handle);
+    std::string object_key = gcp_fh->name;
+    gcp_file_system *fs = reinterpret_cast<gcp_file_handle *>(gcp_fh)->file_system;
+    bool exists;
+    size_t size;
+
+    // Get file size if the object exists.
+    fs->gcp_conn->object_exists(object_key, exists, size);
+
+    sizep = reinterpret_cast<wt_off_t *>(size);
```

Comment:
I think you need to do *sizep

Reply:
addressed here: [687beb0](https://github.com/wiredtiger/wiredtiger/pull/8777/commits/687beb0e754c4e60654dfe49bdeda9dc9bf03665)